misk/src/main/kotlin/misk/web/actions/ReadinessCheckAction.kt

```diff
@@ -18,13 +21,21 @@ class ReadinessCheckAction @Inject internal constructor(
   @Get("/_readiness")
   @ResponseContentType(MediaTypes.APPLICATION_JSON)
   fun readinessCheck(): Response<String> {
-    val isReady = services.all { it.isRunning }
-        && healthChecks.all { it.status().isHealthy }
+    val servicesNotRunning = services.filter { !it.isRunning }
+    val failedHealthChecks = healthChecks
+        .map { it.status() }
+        .filter { !it.isHealthy }
+
+    if (servicesNotRunning.isEmpty() && failedHealthChecks.isEmpty()) {
+      return Response("", statusCode = 200)
+    }
 
-    // TODO(jgulbronson) - Should return an empty body
-    return Response(
-        "",
-        statusCode = if (isReady) 200 else 503
-    )
+    for (service in servicesNotRunning) {
+      logger.error("Service not running: $service")
+    }
+    for (healthCheck in failedHealthChecks) {
+      logger.error("Failed health check: ${healthCheck.messages}")
```

Comment:
it seems like you should get this for free when a `Action` fails. @mmihic thoughts?

Reply:
What should you get for free?

Reply:
if an action fails, misk logs the response for you. that would require adding the messages into the response, which seems reasonable?

Reply:
The current actions do not log on failure, they only return a `HealthStatus`.  We could update `HealthStatus.unhealthy` to automatically log...